set_vcf_sample_id/README.md

```diff
@@ -0,0 +1,185 @@
+# Update a VCF header line sample ID
+
+This pipeline is useful when you have a set of single-sample VCFs
+in Cloud Storage in which the sample ID in the header needs to be changed.
+
+For example, suppose you have a set of 24 VCFs for a single individual
+where each VCF contains variants for a single chromosome (1-22, X, Y). 
+Suppose further that the header line of each VCF looks like:
+
+```
+#CHROM POS ID REF ALT QUAL FILTER INFO FORMAT SAMPLE-001
+```
+
+and you want it to look like:
+
+```
+#CHROM POS ID REF ALT QUAL FILTER INFO FORMAT SAMPLE-001-TEST-01
+```
+
+The scripts in this example can be used to make this change to a set of VCFs.
+The input VCFs can be compressed with gzip or bzip2 or they can be uncompressed.
+The output VCFs' compression state will reflect the input VCFs'.
+
+| API Note |
+|----------|
+| This example demonstrates using the Pipelines API to run custom code *without building a Docker image*. In this example, the scripts are copied into Cloud Storage and then downloaded at pipeline run time.  The docker image used is the stock [Python 2.7 image from dockerhub](https://hub.docker.com/_/python/). |
+
+| API Note |
+|----------|
+| This example demonstrates the use of non-file input parameters. These parameters (`ORIGINAL_SAMPLE_ID` and `NEW_SAMPLE_ID`) get set in the environment and are then available to the code running in the docker container. |
+
+## (1) Copy scripts to Cloud Storage
+
+```
+gsutil cp set_vcf_sample_id.sh set_vcf_sample_id.py \
+  gs://YOUR-BUCKET/pipelines/set_vcf_sample_id/
+```
```

Comment:
This example uses subdirectory `pipelines`.

For the samtools and bioconductor samples, the pattern we've been using is:
```
gs://YOUR-BUCKET/pipelines-api-examples/<particular example name>/
gs://YOUR-BUCKET/pipelines-api-examples/<particular example name>/<scripts here, if applicable>
gs://YOUR-BUCKET/pipelines-api-examples/<particular example name>/output/<output here>
gs://YOUR-BUCKET/pipelines-api-examples/<particular example name>/logging/<logs here>
```

For fastqc:
```
  --output gs://YOUR-BUCKET/fastqc/output/<output here>
  --logging gs://YOUR-BUCKET/fastqc/logging/<logs here>
```

For compress:
```
  --output gs://YOUR-BUCKET/compress/output_path/<output here>
  --logging gs://YOUR-BUCKET/compress/logging_path/<logs here>
```

Let's pick one convention, use it for this sample, and file a bug for the other samples to be updated to the convention.